buildlib/tools/builds.sh

```diff
@@ -448,6 +448,30 @@ then
 			'build_armclang')
 fi
 
+# Check if a specific test was requested as an argument
+if [ "$1" ]; then
+	specific_test=$1
```

Comment:
-> `test_name` (like below)?

Reply:
done

Reply:
Unresolved

Reply:
@yosefe @gleon99 please check now

Reply:
pls rename the variable specific_test to test_name

Reply:
done @yosefe